composer.json

```
@@ -19,38 +19,38 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "^5.2",
-        "symfony/console": "^5.2",
-        "symfony/dotenv": "^5.2",
-        "symfony/expression-language": "^5.2",
+        "symfony/asset": "5.3.0-rc1",
```

Could be: 
```suggestion
        "symfony/asset": "^5.3",
```

Then we dont have to change this again when the 5.3.0 is actually released.